### ISSUE TITLE
[COLOR]: New Feedback button MUST meet minimum WCAG color contrast

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#11905]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The new Feedback button triangle has a yellow background with white text. WCAG 2.0 Level AA requires the visual presentation of text and images of text **must** have a contrast ratio of at least 4.5:1. WCAG 2.1 Level AAA requires the visual presentation of text and images of text has a contrast ratio of at least 7:1.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a low-vision user, I want to clearly understand the content on the site and be able to determine what is interactive by following consistent cues for interactivity.

## Environment

* Operating System: all
* Browser: all
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov in browser
2. On the homepage, there may be a small yellow triangle in the lower right corner

## Possible Fixes (optional)

**Current**
background color: #fdb81e
text color: #fff 
contrast ratio: 1.74

**Possible fix**
background color: #0071bb (also the color used to indicate links)
text color: #fff 
contrast ratio: 5.14

**Current code**
```html
<span id="kampyleButtonContainer">
  <button id="nebula_div_btn" style="border: none; background-image: none; right: 0px; margin-bottom: 0px; background-position: initial initial; background-repeat: initial initial;" alt="Feedback" tabindex="0" class="kampyle_feedback-button kampyle_bottom-right  noOutline ">
    <div class="kampyle_triangle"></div>
    <div data-aut="feedback" class="kampyle_button-text kampyle_bottom">Feedback</div>
  </button>
</span>
```

## WCAG or Vendor Guidance (optional)

* [W3C WCAG Understanding Success Criterion 1.4.3: Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

## Screenshots or Trace Logs

![Screen Shot 2020-07-31 at 4 35 59 PM](https://user-images.githubusercontent.com/57469/89076889-27b0d980-d34f-11ea-9c1b-b1228a769723.png)

![Screen Shot 2020-07-31 at 4 58 55 PM](https://user-images.githubusercontent.com/57469/89076899-2da6ba80-d34f-11ea-8254-f0bb23f3bd7e.png)

**Comment by joanneesteban** [comment]
@jenstrickland can you still see this in Staging after clearing your cache and cookies? This button shouldn't be appearing, and I'm unable to recreate this on my end.

**Comment by jenstrickland** [comment]
I'm not seeing it now. If it's no longer active, we can close the ticket.
